_non-cosmos/namada/chain.json

```diff
@@ -0,0 +1,154 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "namada",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://namada.net",
+  "pretty_name": "Namada",
+  "chain_type": "namada",
+  "chain_id": "namada.5f5de2dd1b88cba30586420",
+  "slip44": 877,
+  "bech32_prefix": "tnam",
+  "daemon_name": "unam",
+  "key_algos": [
+    "ed25519"
+  ],
+  "features": [
```

Comment:
this property is not accepted in the schema

Reply:
done in 558bd76e84ff8585bd6080eea58a52f64a972b3d